Library/src/lab1/Library.java

```diff
@@ -2,48 +2,67 @@
 
 /**
  * A representation of a library of books.
- * @author <a href="mailto:rkhatchadourian@citytech.cuny.edu">Raffi Khatchadourian</a>.
+ *
+ * @author mingjiehuang 
  */
 public class Library {
-    // TODO: Add the missing implementation to this class
 
-    public static void main(String[] args) {
```

Comment:
It seems that you basically spread out the main method into these other methods. There's not much logic here to grade.